releasenotes/releasenotes_v5.rst

```diff
@@ -0,0 +1,164 @@
+#. Searching for a resource with multiple sort fields does not throw an exception anymore.
+#. Fields that are included in the audit event log and AuditEvent resources now contain the same content.
+#. When using the ``If-Modified-Since`` header, only resources that were modified after the specified timestamp are returned. Because of a precision mismatch (seconds vs. milliseconds), wrong resources were sometimes returned before this fix.
+#. When updating a deleted resource conditionally, Firely Server does not throw an exception anymore.
+#. Firely Server now returns the correct issue code when performing a conditional update using ``_id`` as a parameter. Additionally, the error message has been improved when a resource in a different information model is matched via the ``id`` field.
+#. When executing a ``POST``-based search, Firely Server will now return the correct self-link as seen in ``GET``-based searches.
+#. Upon commencing a Bulk Data Export, Firely Server now correctly handles ``Prefer`` headers as outlined `in the specification <https://hl7.org/fhir/uv/bulkdata/export/index.html#headers>`_.
+#. ``Device`` can now be added as an additional resource in a Bulk Data export.
+#. The client id of the default SMART authorization options have been changed from ``vonk`` to ``firelyserver``.
+#. Firely Server now returns improved error messages if the client is not allowed to perform searches.
+#. Support for Firely Server using a SQLite database on M1 Macs was improved. 
+#. During SMART on FHIR v2 discovery, Firely Server now returns the ``grant_types_supported`` field.
+#. Firely Server now returns the correct CodeSystem ``http://terminology.hl7.org/CodeSystem/restful-security-service`` within the security section of its ``CapabilityStatement``. Before this change, the old R3 CodeSystem ``http://hl7.org/fhir/restful-security-service`` was falsely returned.
+#. When performing a Bulk Data Export request with a Firely Server instance running on a SQL database, it will return the Group resource even if it has no members. 
+#. Firely Server will now handle duplicate DLLs and assemblies more gracefully in case they were accidently added to its plugin directory.
+
+Configuration
+^^^^^^^^^^^^^
+#. The configuration section for additional endpoints in the discovery document and additional issuers in tokens has been reworked. Consult the :ref:`SMART Configuration section<feature_accesscontrol_config>` for more details.
+
+Plugin and Facade
+^^^^^^^^^^^^^^^^^
+
+#. Firely Server now uses the `Firely .NET SDK 4.3.0 <https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v4.3.0-stu3>`_. Follow the link for an overview of all changes.
+#. Please note that the ``Vonk.Smart`` package will not be published on NuGet anymore.
+
+The following modules of the public API have either been deprecated, are no longer part of the public API or have been altered in some other way: 
```

Comment:
We should indicate which ones have been removed and which are still available.

Reply:
I improved the wording, as it does not make a difference for developers whether these have been removed or marked as internal. In either way, they are not accessible anymore. What do you think about the changes?

Reply:
Oh sorry. I somehow thought that we moved some namespaces ourselves that would affect the public API. However that's of course not the case. All good.